noggin/form/validators.py

```diff
@@ -0,0 +1,22 @@
+from flask_babel import lazy_gettext as _
+from wtforms.validators import ValidationError, StopValidation
+from wtforms.validators import Email as WTFormsEmailValidator
+
+
+class Email(WTFormsEmailValidator):
+    """ Extend the WTForms email validator, adding the ability to blocklist
+        email addresses
+    """
+
+    def __init__(self, blocklist=None, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.blocklist = blocklist
+
+    def __call__(self, form, field):
+        try:
+            super().__call__(form, field)
+        except ValidationError as e:
+            raise StopValidation(e)
```

Comment:
I don't think it's technically necessary to do this, unless you want validation to stop for validators that would be added *after* the Email validation class in the field's `validators` argument. If the only thing you want is to avoid the `IndexError` caused by the absence of a `@` in the field value, then you can let the `ValidationError` bubble up from the `super()` call.
Both behaviors are fine for me.